flexget/plugins/clients/deluge.py

```diff
@@ -658,10 +660,14 @@ def rename(file, new_name):
                             entry['title'],
                             opts.get('main_file_ratio') * 100))
 
+                if config.get('container_directory'):
+                    log.info('Renaming Folder %s to %s' % (status['name'], pathscrub(entry.render(config.get('container_directory')))))
```

Comment:
Why not just render it once into a variable and then pass that variable to both the logger and the Deluge function? Saves a bit of processing.